spyder/config/manager.py

```diff
@@ -104,7 +105,32 @@ def register_plugin(self, plugin_class):
                 backup=True,
                 raw_mode=True,
                 remove_obsolete=False,
+                external_plugin=True
             )
+
+            # Recreate external plugin configs to deal with part two
+            # (the shortcut confliftcs) of spyder-ide/spyder#11132
+            spyder_config = self._user_config._configs_map['spyder']
+            if check_version(spyder_config._old_version, '54.0.0', '<'):
+                # Remove all previous .ini files
+                try:
+                    plugin_config.cleanup()

```

Comment:
Is this removing the old files?

Reply:
Yes, it is. Since the old external plugin config files are corrupted, I prefer to remove them.